### ISSUE TITLE
WCAG errors in 2.0.0-alpha.2

### ISSUE THREAD

**Issue by Poofy83** [dso-toolkit/dso-toolkit#143]
aanvragen.json,
checken.json
header.json

```json
      {
        "heading": "ERROR",
        "issue": "WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2",
        "description": "This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute \"submit\" or \"image\", or BUTTON elements with type \"submit\" or omitted/invalid.",
        "position": {
          "lineNumber": 1,
          "columnNumber": 0
        },
        "element": {
          "node": "<form class=\"dso-search\">\n      <div class=\"dso-searchbar dso-open\">\n  <input type=\"text\" id=\"searchbar\" class=\"form-control\" placeholder=\"Zoeken\">\n  <span class=\"dso-searchbar-icon\" aria-hidden=\"true\"></span>\n  <label for=\"searchbar\">Zoeken</label>\n  <button type=\"button\">Open zoeken</button>\n</div>\n\n    </form>",
          "class": "dso-search",
          "id": ""
        }
      },
```

inline-group-input.json
```json
      {
        "heading": "ERROR",
        "issue": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Input.Name",
        "description": "This  input element does not have a name available to an accessibility API. Valid names are: element content.",
        "position": {
          "lineNumber": 1,
          "columnNumber": 0
        },
        "element": {
          "node": "<input type=\"\" id=\"group_input_voorbeeld\" class=\"form-control\">",
          "class": "form-control",
          "id": "group_input_voorbeeld"
        }
      },
```

pagination:
* Anchor heeft geen vulling. Teksten "Vorige" en "Volgende" in de anchor zetten maar verbergen met dezelfde techniek als `.sr-only`

**Comment by tfrijsewijk** [comment]
https://www.dso-toolkit.nl/3.0.0-alpha.0/

**Comment by MichielDHVD** [comment]
Voor mij lastig te bepalen of dit naar done kan. Ik heb zelf geen criteria waar ik tegen aan zou houden. Als jij dit als klaar beschouwd mag deze naar Done.
